### PR TITLE
Add missing information to xml documentation for custom functions

### PR DESCRIPTION
**Pull request by jason-chan-lf**
- Also add timeout to workflow jobs

### CODE REVIEW

File: src/Clients/EntriesClient.cs [hunk 1]
```diff
@@ -55,6 +64,9 @@ partial interface IEntriesClient
         /// <exception cref="ApiException">A server side error occurred.</exception>
         Task GetFieldValuesForEachAsync(Func<ODataValueContextOfIListOfFieldValue, Task<bool>> callback, string repoId, int entryId, string prefer = null, bool? formatValue = null, string culture = null, string select = null, string orderby = null, int? top = null, int? skip = null, bool? count = null, int? maxPageSize = null, CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Get the links assigned to an entry using paging. Page results are returned to the <paramref name="callback"/>.
```
Comment (samadpaydar3) [inline src/Clients/EntriesClient.cs]: might be better to use "Returns" to look similar to other ones.

File: src/Clients/EntriesClient.cs [hunk 2]
```diff
@@ -69,6 +81,9 @@ partial interface IEntriesClient
         /// <exception cref="ApiException">A server side error occurred.</exception>
         Task GetLinkValuesFromEntryForEachAsync(Func<ODataValueContextOfIListOfWEntryLinkInfo, Task<bool>> callback, string repoId, int entryId, string prefer = null, string select = null, string orderby = null, int? top = null, int? skip = null, bool? count = null, int? maxPageSize = null, CancellationToken cancellationToken = default);
 
+        /// <summary>
+        /// Get the tags assigned to an entry using paging. Page results are returned to the <paramref name="callback"/>.
```
Comment (samadpaydar3) [inline src/Clients/EntriesClient.cs]: Returns

File: src/Clients/EntriesClient.cs [hunk 3]
```diff
@@ -84,39 +99,43 @@ partial interface IEntriesClient
...
         /// <param name="maxPageSize">Optionally specify the maximum number of items to retrieve.</param>
         /// <param name="cancellationToken">Optional cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
-        /// <returns></returns>
+        /// <returns>Get field values successfully.</returns>
+        /// <exception cref="ApiException">A server side error occurred.</exception>
         Task<ODataValueContextOfIListOfFieldValue> GetFieldValuesNextLinkAsync(string nextLink, int? maxPageSize = null, CancellationToken cancellationToken = default);
 
         /// <summary>
-        /// Get a collection of WEntryLinkInfo.
+        /// Get the links assigned to an entry using a nextlink.
```
Comment (samadpaydar3) [inline src/Clients/EntriesClient.cs]: Returns

File: src/Clients/EntriesClient.cs [hunk 4]
```diff
@@ -84,39 +99,43 @@ partial interface IEntriesClient
...
         /// <param name="maxPageSize">Optionally specify the maximum number of items to retrieve.</param>
         /// <param name="cancellationToken">Optional cancellation token that can be used by other objects or threads to receive notice of cancellation.</param>
-        /// <returns></returns>
+        /// <returns>Get links successfully.</returns>
+        /// <exception cref="ApiException">A server side error occurred.</exception>
         Task<ODataValueContextOfIListOfWEntryLinkInfo> GetLinkValuesFromEntryNextLinkAsync(string nextLink, int? maxPageSize = null, CancellationToken cancellationToken = default);
 
         /// <summary>
-        /// Get a collection of WTagInfo.
+        /// Get the tags assigned to an entry using a nextlink
```
Comment (samadpaydar3) [inline src/Clients/EntriesClient.cs]: Returns

File: src/Clients/FieldDefinitionsClient.cs [hunk 5]
```diff
@@ -4,8 +4,14 @@
 
 namespace Laserfiche.Repository.Api.Client
 {
+    /// <summary>
+    /// The Laserfiche Repository Field Definitions API client.
+    /// </summary>
     partial interface IFieldDefinitionsClient
     {
+        /// <summary>
+        /// Get a collection of field definitions using paging. Page results are returned to the <paramref name="callback"/>.
```
Comment (samadpaydar3) [inline src/Clients/FieldDefinitionsClient.cs]: If you think it's better to use "Returns" instead of "Get", please consider this for the next cases also.

Comment (jason-chan-lf) [reply src/Clients/FieldDefinitionsClient.cs]: I updated all the cases to use "Returns" instead of "Get"

Comment (samadpaydar3) [reply src/Clients/FieldDefinitionsClient.cs]: Thanks.

**Review by samadpaydar3 — APPROVED** [review]

**Review by samadpaydar3 — APPROVED** [review]

**Pull request merged**